platform/broadcom/sonic-platform-modules-inventec/d6556/modules/inv_platform.c

```diff
@@ -0,0 +1,226 @@
+#include <linux/i2c.h>
+//#include <linux/i2c-algo-bit.h>
+#include <linux/i2c-gpio.h>
+#include <linux/init.h>
+#include <linux/module.h>
+#include <linux/slab.h>
+#include <linux/platform_device.h>
+#include <linux/delay.h>
+#include <linux/i2c/pca954x.h>
+//#include <linux/i2c/pca953x.h>
+//#include <linux/i2c/at24.h>
+
+//#include <asm/gpio.h>
+//#define IO_EXPAND_BASE    64
+//#define IO_EXPAND_NGPIO   16
```

Comment:
please remove such commented line in your driver.

Reply:
For transceiver.c, those attributes are only for troubleshooting becasue of some bad transceivers (sfp/qsfp parser in the user space read the whole eeprom for parsing, some bad transceivers have weird issues)

Reply:
unnecessary comments in inv_platform.c is removed. (9111f1d)

Reply:
what was the issue for transceiver? can you help to explain why such troubleshooting cannot be done in user space?

Reply:
Those bad transceiver will hang the i2c bus. The issue could be identified on the user space as well. Those attributes are only for double confirmed the situation. Actually, once the i2c bus dead, we cannot recover it without reboot in this version either. we will figure out how to do it in the next version.